_posts/2022-12-22-release.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Release 2022.51
+author: Priya
+date: 2022-12-22
+category: Jekyll
+layout: post
+---
+
+### Enhancements
+
+#### Show data flow map for views.
+Mammoth now shows the data flow map at view level.It shows how data is transformed as it flows through different views within the organization.
+
+#### Allow duplication of view as well as pipelines upon ds duplication.
+When a user duplicates a data source,now they can choose to duplicate the data itself as well as any views and pipelines associated with the data source. This can save time and effort by allowing users to quickly create copies of existing data sources and their associated views and pipelines.
+
+
+### Defect fixes
+
+* Clicking on the mammoth icon takes the user to the root folder.
+* Reset the view throws incorrect notification. 
+
+
+### API modifications
```

Comment:
_We can remove this section as it is quite internal and not related to Mammoth APIs_